assets/kube-controller-manager/kube-controller-manager-deployment.yaml

```diff
@@ -70,6 +70,38 @@ spec:
         args:
         - "--openshift-config=/etc/kubernetes/cmconfig/config.yaml"
         - "--kubeconfig=/etc/kubernetes/secret/kubeconfig"
+        - "--authentication-kubeconfig=/etc/kubernetes/secret/kubeconfig"
```

Comment:
Should these be removed from `assets/kube-controller-manager/config.yaml` as well?